src/cls/lock/cls_lock_client.h

```diff
@@ -73,18 +83,28 @@ namespace rados {
 	std::string description;
 	utime_t duration;
 	uint8_t flags;
+    int32_t bid_amount;
+	utime_t bid_duration;
 
       public:
 
-	Lock(const std::string& _n) : name(_n), flags(0) {}
-
+	Lock(const std::string& _n) : name(_n),
+                                  flags(0),
+                                  bid_amount(-1),
+                                  bid_duration(utime_t())
```

Comment:
nit: this is the same as default-constructing `bid_duration`, so we can remove it from the constructor's member initializer list entirely